test/odbc/psqlodbc/psqlodbc_tests_common.cpp

```diff
@@ -0,0 +1,886 @@
+#include "psqlodbc_tests_common.h"
+
+// helper function to initialize insert string (1, "", "", ""), etc.
+string InitializeInsertString(const vector<string> &insertedValues, bool isNumericInsert) {
+
+  string insertString{};
+  string comma{};
+  if (isNumericInsert) {
+    for (int i = 0; i < insertedValues.size(); i++) {
+      insertString += comma + "(" + std::to_string(i) + "," + insertedValues[i] + ")";
+      comma = ",";
+    }
+    return insertString;
+  }
+
+  for (int i = 0; i < insertedValues.size(); i++) {
+    if (insertedValues[i] == "NULL") {
+      insertString += comma + "(" + std::to_string(i) + "," + insertedValues[i] + ")";
+      comma = ",";
+    }
+    else {
+      insertString += comma + "(" + std::to_string(i) + "," + "'" + insertedValues[i] + "'" + ")";
+      comma = ",";
+    }
+  }
+  return insertString;
+}
+
+
+void testCommonColumnAttributes(string tableName, vector<pair<string, string>> tableColumns, string orderByColumnName, vector<int> lengthExpected, vector<int> precisionExpected, vector<int> scaleExpected, vector<string> nameExpected) {
+  
+  OdbcHandler odbcHandler(Drivers::GetDriver(ServerType::PSQL));
+
+  const int NUM_COL = tableColumns.size();
+  const int BUFFER_SIZE = 256;
```

Comment:
Stuff like `BUFFER_SIZE`, `BUFFER_LENGTH`, and `CHARSIZE` Can be made global

Reply:
Done. They are located in `psqlodbc_tests_common.h`